app/scripts/modules/core/src/pipeline/triggers/NextRunTag.tsx

```diff
@@ -27,7 +27,8 @@ export class NextRunTag extends React.Component<INextRunTagProps, INextRunTagSta
 
   private updateSchedule(): INextRunTagState {
     if (this.props.pipeline) {
-      const crons = (this.props.pipeline.triggers || []).filter(t => t.type === 'cron' && t.enabled) as ICronTrigger[];
+      const crons = (this.props.pipeline.triggers || [])
+        .filter((t: ICronTrigger) => t.type === 'cron' && t.enabled && t.cronExpression) as ICronTrigger[];
```

Comment:
this is not at all related to the title of the PR but it was sitting there in my local changes and it's a good change